src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LocalAccess.cs

```diff
@@ -417,14 +419,16 @@ internal MutableBox(int index, Type type)
             }
 
             [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2077:UnrecognizedReflectionPattern",
-                Justification = "_type is a ValueType. You can always create an instance of a ValueType.")]
+                Justification = "_type is a ValueType. You can always get an uninitialized ValueType.")]
             public override int Run(InterpretedFrame frame)
             {
                 object? value;
 
                 try
                 {
-                    value = Activator.CreateInstance(_type);
+                    value = _type.IsNullableType() ?
+                        Activator.CreateInstance(_type) :
+                        RuntimeHelpers.GetUninitializedObject(_type);
```

Comment:
This pattern is repeated, did you consider adding a helper?

Reply:
Will leave as is to avoid moving suppression messages, etc.